pycrostates/preprocessing/extract_gfp_peaks.py

```diff
@@ -103,6 +117,18 @@ def extract_gfp_peaks(
     picks_all = _picks_to_idx(inst.info, inst.ch_names, none="all", exclude="bads")
     _check_picks_uniqueness(inst.info, picks)
 
+    # gfp function
+    ch_type = inst.get_channel_types(picks)[0]
```

Comment:
Is it safe to assume channel type uniqueness in the dataset? If so, I would write this as:

```suggestion
    ch_type = inst.get_channel_types(picks, unique=True)
    assert len(ch_type) == 1  # sanity-check
    ch_type = ch_type[0]
```

Reply:
Yes it is safe due to `_check_picks_uniqueness(inst.info, picks)`